requirements.txt

```diff
@@ -5,4 +5,4 @@ Logbook
 python-hglib
 pytoml
 structlog
-taskcluster
+taskcluster==24.0.0
```

Comment:
Can we do >=? Or not specify it at all?

Reply:
I prefer to explicitly set the min version, will switch to `>=`